alpha/apps/kaltura/lib/batch2/kFlowManager.php

```diff
@@ -493,25 +493,30 @@ public function objectAdded(BaseObject $object, BatchJob $raisedJob = null)
 			{
 				if ($entry->getType() == entryType::MEDIA_CLIP)
 				{
-					$syncKey = $object->getSyncKey(flavorAsset::FILE_SYNC_FLAVOR_ASSET_SUB_TYPE_ASSET);
-
-					if (kFileSyncUtils::fileSync_exists($syncKey))
+					if ($entry->getOperationAttributes() && $object->getIsOriginal())
```

Comment:
Please use $entry->getDynamicFlavorAttributesForAssetParams instead of $entry->getOperationAttributes, only operation attributes on the source should matter.